content/docs/started/getting-started.md

```diff
@@ -16,34 +22,61 @@ It's important that you have some knowledge of the following systems and tools:
 [Kubernetes system requirements](/docs/started/k8s/overview#minimum-system-requirements).
 
+## Overview of installation options
+
+The following diagram gives an overview of the options for deploying Kubeflow:
+
+<img src="/docs/images/kubeflow-getting-started-diagram.svg" 
+  alt="A diagrammatic overview of Kubeflow deployment options"
+  class="mt-3 mb-3 border border-info rounded">
+
+The following section describes the options in more detail and links to the
+relevant instructions.
+
 <a id="installation-guides"></a>
 ## Installing Kubeflow
 
 There are various ways to install Kubeflow. Choose one of the following options
-to suit your environment (desktop or server, existing Kubernetes cluster, or public cloud):
+to suit your environment (public cloud, existing Kubernetes cluster, or
+desktop or server).
 
-* Installing Kubeflow on a **desktop** or **server**:
+### Installing Kubeflow on a public cloud
 
-  * To use Kubeflow on Windows,
-  follow the [Windows deployment guide](/docs/started/workstation/getting-started-windows/).
-  * To use Kubeflow on MacOS,
-  follow the [MacOS deployment guide](/docs/started/workstation/getting-started-macos/).
-  * To use Kubeflow on Linux,
-  follow the [Linux deployment guide](/docs/started/workstation/getting-started-linux/).
+Choose the Kubeflow deployment guide for your chosen cloud:
 
-* Installing Kubeflow on a **existing Kubernetes cluster** or a **public cloud**:
-
-  * To install Kubeflow on a Kubernetes cluster, follow the
-  [guide to deploying Kubeflow on Kubernetes](/docs/started/k8s/overview/).
   * To use Kubeflow on Google Cloud Platform (GCP) and Kubernetes Engine (GKE),
-  follow the [GCP deployment guide](/docs/gke/deploy/). To use MiniKF (mini
-  Kubeflow) on GCP, follow the [MiniKF on GCP guide](/docs/started/workstation/minikf-gcp/).
+    follow the [GCP deployment guide](/docs/gke/deploy/).
   * To use Kubeflow on Amazon Web Services (AWS),
-  follow the [AWS deployment guide](/docs/aws/deploy/).
+    follow the [AWS deployment guide](/docs/aws/deploy/).
   * To use Kubeflow on Microsoft Azure Kubernetes Service (AKS),
-  follow the [AKS deployment guide](/docs/azure/deploy/).
+    follow the [AKS deployment guide](/docs/azure/deploy/).
   * To use Kubeflow on IBM Cloud Private (ICP),
-	follow the [ICP deployment guide](/docs/started/cloud/getting-started-icp/).
+	  follow the [ICP deployment guide](/docs/started/cloud/getting-started-icp/).
+
+### Installing Kubeflow on an existing Kubernetes cluster
+
+Follow the
+[guide to deploying Kubeflow on Kubernetes](/docs/started/k8s/overview/).
+
+### Installing Kubeflow on a desktop or server
+
+{{% alert title="<b>Not</b> recommended for getting started with Kubeflow" 
+  color="info" %}}
+The procedures for installing Kubeflow on a desktop or server can be complex and
+highly dependent on your environment. The current recommendation is to choose
+a cloud installation if you're just getting started with Kubeflow.
```

Comment:
Nit: Remove "just"

The Google developer documentation style recommends against using  "just" since it can generally be removed from sentences without affecting the meaning.

Reply:
Thanks Joe! Style guides are worth following. :) However, in this case, I think the word "just" does add meaning. I tried removing it, but "if you're getting started with Kubeflow" carries different connotations from "if you're just getting started with Kubeflow". The first could mean you're well along the way, whereas the second means you're just starting out. Another option would be to reword the entire sentence, but I've decided not to go that far.